### DISCUSSION TITLE
Component Testing: Nextjs - Cannot GET /__cypress/src/index.html

### DISCUSSION THREAD

**Discussion by manoj-mukherjee-maersk** [cypress-io/cypress#22315] [Questions and Help] [answered]
Getting this message - Cannot GET /__cypress/src/index.html when trying run button.cy.jsx component Testing. What need to do? 

Framework: Nextjs -   `"next": "^12.1.0"`

node_module css in compoent.js -> import "componentlib/dist/lib/css/global.css";

ERROR in ./node_modules/componentlib/dist/lib/css/global.css
No template for dependency: ModuleHotAcceptDependency

```js
component: {
    devServer: {
      framework: "next",
      bundler: "webpack",
    },
  },
```
```bash
Cypress Version: 10.1.0 (stable)
System Platform: darwin (21.3.0)
```

<img width="1421" alt="Screenshot 2022-06-14 at 10 49 06 PM" src="https://user-images.githubusercontent.com/88817537/173638273-a62fed25-1be2-40cd-824b-4a83f475de20.png">

**Comment by manoj-mukherjee-maersk** [comment, marked as answer]
It's fix now. I follow this step
- Install `"next-global-css": "^1.3.1"` and follows their usage.
- Create global.css in `src/cy-setup/global.css` and import lib css in global.css file. 
- import that in support/component.js file

**Comment by paul-moers** [comment]
Since Cypress 10 it might also be when you have your files in a different location, then you now also need to specify `indexHtmlFile: 'path/to/your/component-index.html'`.

**Reply by dwhieb** [reply]
I get this same error even when I have the component HTML file specified in the `indexHtmlFile` option. In my case I'm using a custom dev server.
